docs/reference/module-api/module-overview.md

```diff
@@ -933,17 +930,17 @@ Task functions takes the object `argv` as an argument, which includes the argume
 > Input:
 >
 > ```
-> node app jump sideways --foo --bar=whee --super-cool=totally
+> node app my-module:runIt taskOption --foo --bar=baz --use-color=green
 > ```
 >
 > Response:
 >
 > ```
 > {
->   _: [ "jump", "sideways"],
+>   _: [ "runIt", "taskOption"],
```

Comment:
It'll be `my-module:runIt`, not just `runIt`. It's the actual command line arguments from `boring` (everything after app.js), the task module doesn't try to get cute with it in any way.